masterlist.yaml

```
@@ -8741,6 +8741,53 @@ plugins:
     after: [ 'SkyTEST-RealisticAnimals&Predators.esp' ]
     tag: [ Text ]
 
+  - name: 'Apothecary.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/52130/' ]
+    after: [ 'TechnicolorAlchemy.esp' ]
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Beyond Skyrim - Bruma' ]
+        condition: 'active("BSAssets.esm") or active("BSHeartland.esm") and not active("Apothecary - Bruma Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Creation Club Fishing' ]
+        condition: 'active("ccBGSSSE001-Fish.esm") and not active("Apothecary - Fishing Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Creation Club Rare Curios' ]
+        condition: 'active("ccbgssse037-curios.esl") and not active("Apothecary - Rare Curios Patch.esp")'
```

This is more of an aesthetic suggestion, but the capitalization of a plugin in a condition would ideally match that of the plugin:

```suggestion
        condition: 'active("ccBGSSSE037-Curios.esl") and not active("Apothecary - Rare Curios Patch.esp")'
```